src/analytics/js/honeycomb.analytics.google.js

```diff
@@ -60,6 +65,11 @@ let initAccount = ( accountId ) => {
 
     if ( sites ) {
         window.ga( 'create', accountId, 'auto', { 'allowLinker': true } );
+
```

Comment:
Shouldn't this be outside of the `sites` condition? Maybe place it below the whole condition?

Reply:
ah you're right, that does make more sense - done!

Reply:
👍